### PR TITLE
Added boolean value support

### PR DESCRIPTION
**Pull request by x87-va**

### CODE REVIEW

File: src/parser.rs [hunk 1]
```diff
@@ -233,9 +241,10 @@ impl FromStr for CypherValue {
 impl Display for CypherValue {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            CypherValue::Float(float) => write!(f, "{}", float),
-            CypherValue::Integer(integer) => write!(f, "{}", integer),
-            CypherValue::String(string) => f.pad(string),
+            CypherValue::Float(float)       => write!(f, "{}", float),
+            CypherValue::Integer(integer)   => write!(f, "{}", integer),
+            CypherValue::String(string)     => f.pad(string),
+            CypherValue::Boolean(boolean)   => write!(f, "{}", boolean),
```
Comment (s1ck) [inline src/parser.rs:247]: nice alignment on the indentation! is that what `cargo fmt` produces?

File: src/parser.rs [hunk 2]
```diff
@@ -302,8 +311,15 @@ fn string_literal(input: &str) -> IResult<&str, CypherValue> {
     )(input)
 }
 
+fn boolean_literal(input: &str) -> IResult<&str, CypherValue> {
+  alt((
+      map(tag("false"), |_| CypherValue::Boolean(false)),
+      map(tag("true"), |_| CypherValue::Boolean(true))
```
Comment (s1ck) [inline src/parser.rs]: maybe we could use `nom::tag_no_case` instead of `tag`?

**Review by s1ck — APPROVED** [review]
Thanks for the PR! I like it, my only comment is to maybe replace `nom::tag` with `nom::tag_no_case`. But we can also merge it as is, if you don't want to change it.

File: src/parser.rs (hunk 2 above)
Comment (x87-va) [reply src/parser.rs]: Yes, of course.

File: src/parser.rs (hunk 1 above)
Comment (x87-va) [reply src/parser.rs:247]: No, it was made manually.

**Comment by x87-va** [comment]
I've fixed literals to be case insensitive.

**Comment by s1ck** [comment]
Looks good! Thanks for the contribution :)

**Pull request merged**